### PR TITLE
GV2 - 65 Icons

### PR DESCRIPTION
**Pull request by AlinaKuzmenko**

### CODE REVIEW

File: src/components/Icons/types.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { SVGProps } from 'react';
+
+export type TPathDProp = 'd';
```
Comment (vladstart1) [inline src/components/Icons/types.ts:3]: Probably you don't need this type

**Review by vladstart1 — DISMISSED** [review]

File: src/components/Icons/IconsList/SocialIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+  ),
+  tikTok: ({ fill, stroke, width = 24, height = 24, ...rest }: React.SVGProps<SVGSVGElement>) => (
+    <svg width={width} height={height} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...rest}>
+      {!fill
+        ? (<>
+          <path
+            d="M17.1765 8.66347C18.7198 9.77064 20.6105 10.4221 22.6525 10.4221V6.47861C22.2661 6.47869 21.8806 6.43825 21.5025 6.35786V9.46193C19.4606 9.46193 17.5702 8.81049 16.0265 7.7034V15.7509C16.0265 19.7766 12.7745 23.0399 8.76333 23.0399C7.26666 23.0399 5.87556 22.5858 4.71997 21.807C6.03888 23.1604 7.87821 24 9.91309 24C13.9245 24 17.1766 20.7367 17.1766 16.7108V8.66347H17.1765ZM18.5951 4.68499C17.8064 3.82023 17.2885 2.70267 17.1765 1.46717V0.959961H16.0867C16.361 2.53028 17.2966 3.87187 18.5951 4.68499ZM7.25706 18.7178C6.81638 18.1379 6.57824 17.4286 6.57931 16.6992C6.57931 14.858 8.06672 13.3651 9.90177 13.3651C10.2438 13.365 10.5837 13.4176 10.9096 13.5214V9.48977C10.5287 9.43739 10.1443 9.41515 9.7601 9.4233V12.5613C9.43394 12.4575 9.09384 12.4048 8.75177 12.4051C6.91671 12.4051 5.42938 13.8979 5.42938 15.7393C5.42938 17.0414 6.17284 18.1687 7.25706 18.7178Z"
+            fill={fill || '#FF004F'}
```
Comment (vladstart1) [inline src/components/Icons/IconsList/SocialIcon.tsx:173]: Probably should be just `fill="#FF004F"` as you specified !fill in ternary. The same applies for two `<path/>` below this

File: src/components/Icons/IconsList/SocialIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+  ),
+  youTube: ({ fill = '#FF0302', stroke, width = 24, height = 24, ...rest }: React.SVGProps<SVGSVGElement>) => (
+    <svg width={width} height={height} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...rest}>
+      <g clipPath="url(#clip0_4115_31798)">
+        <path
+          d="M23.5221 6.18541C23.3864 5.67482 23.119 5.20883 22.7466 4.83407C22.3743 4.4593 21.91 4.18891 21.4003 4.04996C19.5239 3.54541 12.0239 3.54541 12.0239 3.54541C12.0239 3.54541 4.52393 3.54541 2.64756 4.04996C2.13786 4.18891 1.67358 4.4593 1.30121 4.83407C0.928842 5.20883 0.661431 5.67482 0.525744 6.18541C0.0239258 8.06996 0.0239258 12 0.0239258 12C0.0239258 12 0.0239258 15.93 0.525744 17.8145C0.661431 18.3251 0.928842 18.7911 1.30121 19.1658C1.67358 19.5406 2.13786 19.811 2.64756 19.95C4.52393 20.4545 12.0239 20.4545 12.0239 20.4545C12.0239 20.4545 19.5239 20.4545 21.4003 19.95C21.91 19.811 22.3743 19.5406 22.7466 19.1658C23.119 18.7911 23.3864 18.3251 23.5221 17.8145C24.0239 15.93 24.0239 12 24.0239 12C24.0239 12 24.0239 8.06996 23.5221 6.18541Z"
+          fill={fill}
+        />
+        <path d="M9.56934 15.5687V8.4314L15.8421 12L9.56934 15.5687Z" fill="#FEFEFE" />
```
Comment (vladstart1) [inline src/components/Icons/IconsList/SocialIcon.tsx:86]: YouTube icon should have transparent background when filled

File: src/components/Icons/IconsList/SocialIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+        fill={fill}
+      />
+    </svg>
+  ),
+  snapChat: ({ fill = 'black', stroke, width = 24, height = 24, ...rest }: React.SVGProps<SVGSVGElement>) => (
+    <svg width={width} height={height} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" {...rest}>
+      <path
```
[the remark this answers is not in the document]
Comment (AlinaKuzmenko) [reply src/components/Icons/IconsList/SocialIcon.tsx:143]: thanks!

File: src/components/Icons/IconsList/SocialIcon.tsx (hunk 3 above)
Comment (AlinaKuzmenko) [reply src/components/Icons/IconsList/SocialIcon.tsx:86]: thanks

**Review by vladstart1 — APPROVED** [review]

**Pull request merged**